src/main/java/com/google/devtools/build/lib/syntax/StringModule.java

```diff
@@ -274,21 +274,21 @@ public String strip(String self, Object charsOrNone) {
             type = String.class,
             doc = "The string to replace with."),
         @Param(
-            name = "maxsplit",
+            name = "count",
             type = Integer.class,
             noneable = true,
             defaultValue = "None",
             doc = "The maximum number of replacements.")
       })
-  public String replace(String self, String oldString, String newString, Object maxSplitO)
+  public String replace(String self, String oldString, String newString, Object count)
       throws EvalException {
-    int maxSplit = Integer.MAX_VALUE;
-    if (maxSplitO != Starlark.NONE) {
-      maxSplit = Math.max(0, (Integer) maxSplitO);
+    int maxReplaces = Integer.MAX_VALUE;
+    if (count != Starlark.NONE && (Integer) count > -1) {
```

Comment:
Super nit: I feel like `count >= 0` is more readable than `count > -1`.

Reply:
That's not a super nit. >=0 simply *is* more readable than > -1.